src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -1018,3 +1020,17 @@ Box2f CropWindowTool::resolutionGate() const
 	}
 	return resolutionGate;
 }
+
+bool CropWindowTool::keyPress( const KeyEvent &event )
+{
+	if( const auto hotkey = Gaffer::Metadata::value<StringData>( this, "viewer:shortCut" ) )
+	{
+		if( event.key == hotkey->readable() && event.modifiers == KeyEvent::Modifiers::Alt )
+		{
+			const bool newState = !activePlug()->getValue();
+			activePlug()->setValue( newState );
+			m_cropWindowEnabledPlug->setValue( newState );
```

Comment:
This crashes if `m_cropWindowEnabledPlug` is null, which it can be if the tool hasn't found a plug to edit.

Reply:
I protected against that crash in https://github.com/GafferHQ/gaffer/pull/5518/commits/3a8c2fdbf2bd4dd0d3a1f34e8f51ec6e5651ac07